### DISCUSSION TITLE
ERROR: Could not build wheels for cpprb

### DISCUSSION THREAD

**Discussion by tmorstyn** [ymd-h/cpprb#35] [Q&A]
Hi I am trying to install cpprb on a macbook (macOS 13.4.1, Intel Core i9).
Running "pip install cpprb" I receive the error "ERROR: Could not build wheels for cpprb, which is required to install pyproject.toml-based projects". (The current numpy version is 1.26.4). I've also tried the instructions for "Install from source code" which gives the same error.

**Comment by ymd-h** [comment]
Hi, @tmorstyn 

Can you give us the following information?
- Python version
- pip version
- full error messages of `pip -vvv install cpprb`

**Reply by ymd-h** [reply]
With an additional investigation, I found CI jobs silently failed to upload macOS / Windows binary for cpprb-11.0.0.
https://github.com/ymd-h/cpprb/actions/runs/9348919762/job/25729126488

And I manually uploaded these binary wheels.
Perhaps you can install cpprb with those wheel.

**Comment by tmorstyn** [comment]
Thanks, much appreciated. I was now able to install it.
